app/views/courses/show.html.haml

```diff
@@ -39,15 +39,15 @@
       - if user_signed_in? && !@enrollment.nil? && @enrollment.leader?
         .row
           .col.s12.center
-            %a#up{ href: course_lesson_path(@course, lesson, lesson_no: "up"), "data-method" => "patch", class: "up" }
+            %a#up.hoverable{ href: course_lesson_path(@course, lesson, lesson_no: "up"), "data-method" => "patch", class: "up" }
               %i.material-icons expand_less
-            %li= link_to "Lesson #{lesson.lesson_no}: #{lesson.title}", course_lesson_path(@course, lesson)
-            %a#down{ href: course_lesson_path(@course, lesson, lesson_no: "down"), "data-method" => "patch", class: "down" }
+            %li.card-panel= link_to "Lesson #{lesson.lesson_no}: #{lesson.title}", course_lesson_path(@course, lesson)
+            %a#down.hoverable.highlight{ href: course_lesson_path(@course, lesson, lesson_no: "down"), "data-method" => "patch", class: "down" }
```

Comment:
Line is too long. [144/80]<br>Avoid defining `class` in attributes hash for static class names<br>Classes should be listed before IDs (.hoverable should precede #down)